flood_api/settings.py

```diff
@@ -11,6 +11,7 @@ class Settings(BaseSettings):
     summary_data_path: str = "s3://databricks-data-openepi/glofas/processed/newest/processed_summary_forecast.parquet/"
     threshold_data_path: str = "s3://databricks-data-openepi/glofas/auxiliary-data/processed_thresholds.parquet/"
     api_root_path: str = "/"
+    api_description: str = 'This is a RESTful service that provides accurate and up-to-date flood information for the geographic region bounded by the following coordinates: min_lon=-18.0, min_lat=-6.0, max_lon=52.0, max_lat=17.0. <br/>The data are sourced from <a href="https://cds.climate.copernicus.eu/cdsapp#!/dataset/cems-glofas-forecast?tab=overview">https://cds.climate.copernicus.eu/cdsapp#!/dataset/cems-glofas-forecast?tab=overview</a>. <br/>The data are freely available for use under the <a href="https://cds.climate.copernicus.eu/api/v2/terms/static/cems-floods.pdf">CEMS-FLOODS datasets licence</a>.'
```

Comment:
The field `api_desctiption` need to be included when creating the FastAPI-object, otherwise it will not be used in the swagger-documentation.